### PR TITLE
Add payment source to docs

### PR DESCRIPTION
**Pull request by mbbertino**
this adds documentation support for the new `payment_source` field

### CODE REVIEW

File: liquid/variables.md [hunk 1]
```diff
@@ -46,8 +48,10 @@
 - `payment.fee` - the fee in cents
 - `payment.amount_refunded` - the amount refunded in cents
 - `payment.status` - payment status in the dashboard
-- `payment.card.last4` - last 4 digits of the card used
-- `payment.card.type` - the card type (Visa, MasterCard,etc.)
+- `payment.payment_source.type` - Method of payment used, ie Credit/Debit card or Bank Account
```
Comment (thomas07vt) [inline liquid/variables.md]: I think card should be capitalized here right?

Comment (thomas07vt) [reply liquid/variables.md]: Also `ie` should be `i.e.` for consistency

File: liquid/variables.md [hunk 2]
```diff
@@ -14,9 +14,11 @@
 - `plan.payer_manage_url` - The link for the payer to manage their plan
 - `plan.name` - Payer name
 - `plan.email` - Payer email
-- `plan.card.last4` - Last four digits of card
-- `plan.card.type` - Card Type, i.e. Visa, MasterCard, etc.
-- `plan.card.expiration` - The month and year the active card expires
+- `plan.payment_source.type` - Method of payment used, ie Credit/Debit card or Bank Account
```
Comment (thomas07vt) [inline liquid/variables.md]: `ie` should be `i.e.` for consistency I think

Comment (thomas07vt) [inline liquid/variables.md]: Card should be capitalized here I think

File: api/v1/payments.md [hunk 3]
```diff
@@ -62,6 +66,8 @@ Notes:
 * `customer_reference` will be present if the payment was attached to a Stripe customer (vs. a one time payment). Otherwise, it will be null.
 * `invoice_reference` will be present if the payment was generated from a Stripe invoice (vs. a one off payment). Otherwise, it will be null.
 * The `custom_id` field is populated if you use one of our [integration methods](https://github.com/moonclerk/developer/blob/master/integration.md).
+* `brand`, `exp_month` and `exp_year` are only present when the payment method used is a card
+* `bank_name`is only present when the payment method used is a ach
```
Comment (thomas07vt) [inline api/v1/payments.md]: ~is a ach~ is ach

File: api/v1/payments.md [hunk 4]
```diff
@@ -62,6 +66,8 @@ Notes:
 * `customer_reference` will be present if the payment was attached to a Stripe customer (vs. a one time payment). Otherwise, it will be null.
 * `invoice_reference` will be present if the payment was generated from a Stripe invoice (vs. a one off payment). Otherwise, it will be null.
 * The `custom_id` field is populated if you use one of our [integration methods](https://github.com/moonclerk/developer/blob/master/integration.md).
+* `brand`, `exp_month` and `exp_year` are only present when the payment method used is a card
```
Comment (thomas07vt) [inline api/v1/payments.md]: I think `is card` reads better than `is a card`

File: api/v1/customers.md [hunk 5]
```diff
@@ -98,6 +102,8 @@ Notes:
 * Custom field keys are configured in the *Additional Information* section of the payment form builder
 * Any JSON key in the customer object ending in `_reference` corresponds to a related Stripe ID
 * The `custom_id` field is populated if you use one of our [integration methods](https://github.com/moonclerk/developer/blob/master/integration.md).
+* `brand`, `exp_month` and `exp_year` are only present when the payment method used is a card
```
Comment (thomas07vt) [inline api/v1/customers.md]: see comment below

File: api/v1/customers.md [hunk 6]
```diff
@@ -98,6 +102,8 @@ Notes:
...
+* `bank_name`is only present when the payment method used is a ach
```
Comment (thomas07vt) [inline api/v1/customers.md]: see comment below

**Comment by mbbertino** [comment]
@thomas07vt good catches. Changes made.

File: liquid/examples.md [hunk 7]
```diff
@@ -71,5 +71,5 @@ Expiration: {{ plan.expires | date: '%b %d, %Y' }}{% endif %}{% if plan.next_pay
 Next Payment Attempt: {{ plan.next_payment_attempt | date: '%b %d, %Y'  }}{% endif %}
 
 To update your card on file, please visit:
-{{ plan.card.update_url }}{% endif %}
-```
\ No newline at end of file
+{{ plan.payer_manage_url }}{% endif %}
```
Comment (ryanwood) [inline liquid/examples.md]: Are you sure about this one?

**Comment by ryanwood** [comment]
@mbbertino Does this need to get merged?

**Comment by mbbertino** [comment]
Yeah, I think so. (Sorry I must’ve forgotten that)

Matt Bertino

Developer, MoonClerk

**Pull request merged**